Update jib-gradle-plugin/CHANGELOG for the new option: `configurationName` (#3111)
GoogleContainerTools/jib: jib-gradle-plugin/CHANGELOG.md

### Added

- Added an option `configurationName` to specify the name of the [Gradle Configuration](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.ConfigurationContainer.html) to use. The option can be lazily configured, for example, using Gradle `Provider` or `Property`. ([#3034](https://github.com/GoogleContainerTools/jib/pull/3034))
```gradle
    jib {

>       configurationName.set = 'myconfig'

Reviewer: Ah, my bad. I think it's just
```suggestion
      configurationName = 'myconfig'
```
Right? (I think it worked in this way.)

Author: My bad, thank you. Just did a quick test and `configurationName = 'myConfig'` works.

> - Added an option `configurationName` to specify the name of the [Gradle Configuration](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.Configuration.html) to use. This change introduces lazy evaluation by exposing the Gradle Property through a getter, which is different from the traditional route of using Gradle Provider and Property. ([#3034](https://github.com/GoogleContainerTools/jib/pull/3034))
>   ```gradle
>   jib {
>     configurationName.set("myconfig")
>   }
>   ```

Reviewer: I'm pretty sure you can do `configurationName = 'myconfig'`, but you can double-check. How about this?

```suggestion
- Added an option `configurationName` to specify the name of the [Gradle Configuration](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.Configuration.html
) to use. The option can be lazily configured, for example, using Gradle `Provider` or `Property`. ([#3034](https://github.com/GoogleContainerTools/jib/pull/3034))
  ```gradle
  jib {
    configurationName.set = 'myconfig'
  }
  ```
```
